VideoWeb/VideoWeb/ClientApp/src/app/vh-officer/hearings/vho-hearings.component.ts

```diff
@@ -220,6 +220,9 @@ export class VhoHearingsComponent implements OnInit, OnDestroy {
     const participantToUpdate = this.participants.find(x => x.id === message.participantId);
     if (participantToUpdate) {
       participantToUpdate.status = message.status;
+      if (participantToUpdate.role === UserRole.Judge) {
+        this.getJudgeStatusDetails();
```

Comment:
Why are you treating judges differently? What is this method for? Judges already are available in the participant list, status updates shouldn't be any different for non judge participants

Reply:
You did not read a story, judge is treated differently.

Reply:
Could you please give me a proper answer? Why does this method exist? If it's to show the judge's status in another hearing this is not doing so. At what point do you show the judge is in another hearing?

Reply:
@valentinabennett  Is this to show that the judge is in another hearing? This does not apply to participants and hence the code here @shaed-parkar 
 const judgeStatus = this.findJudgeInAnotherHearing(x.participants, selectedJudgeUserName);

Reply:
If judge returns to hearing list, need to change the status on the selected hearing (admin) to Uavailable, so this method do it.

Reply:
I would have advised to do this in the MVC. There's an enum called 'ParticipantStatus' specific to the FE. We could add a status called 'InAnotherHearing'.

Rather than having all this logic in the angular app, the view model could hold this information keeping the angular components leaner. Those components are getting larger and doing far too much now (i.e. applying filters and working out participant statuses).

It's simply a suggestion but we should review the role of the VHO component and how much it is doing at the moment.

Reply:
Think this is ok Shaed. There is a wholesale re-design of this coming in soon.